src/middle/Stan_math_signatures.ml

```diff
@@ -533,35 +482,6 @@ let string_operator_to_stan_math_fns str =
   | _ -> str
 
 (* -- Querying stan_math_signatures -- *)
-let query_stan_math_mem_pattern_support (name : string) (args : fun_arg list) =
```

Comment:
The comment seems to be referring to the deleted/moved function. Is it still relevant?

Reply:
Nope, good catch